_data/hosting.yml

```
@@ -23,6 +23,20 @@ websites:
       - totp
     doc: https://docs.acquia.com/acquia-cloud/access/signin/
 
+  - name: ACTIVE 24
+    url: https://www.active24.com/
+    img: active24.png
+    tfa:
+      - totp
+    doc: https://faq.active24.com/cz/183996-Ov%C4%9B%C5%99en%C3%AD-p%C5%99ihl%C3%A1%C5%A1en%C3%AD---dvoufaktorov%C3%A1-autorizace-2FA?l=cs
+
+  - name: ACTIVE 24 CZ
+    url: https://www.active24.cz/
+    img: active24.png
+    tfa:
+      - totp
+    doc: https://faq.active24.com/cz/183996-Ov%C4%9B%C5%99en%C3%AD-p%C5%99ihl%C3%A1%C5%A1en%C3%AD---dvoufaktorov%C3%A1-autorizace-2FA?l=cs
+
```

I don't think there should be multiple entries, but region specifiers instead.

The fact that there is no doc page in English yet is unfortunate. However, we can at least shorten the URL.

```suggestion
    doc: https://faq.active24.com/cz/183996-
    regions:
      - cz
      - gb
      - nl
      - sk
      - de
      - es

```